### DISCUSSION TITLE
[Feature] <Default volume setting for videos in webui>

### DISCUSSION THREAD

**Discussion by curtwagner1984** [immich-app/immich#10088] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great if the volume you set in one of the videos you've shown will be the default volume everywhere. Or alternativly if you could set the default volume anywhere. It's quite annoying when setting slideshow as a background, and randomly stumbling into a video with screaming children. 

I wish I was able to mute all videos by default for example. I searched for the feature in the webui but didn't find it, I also searched in the issue tracker, didn't find this feature requrest either.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by michelheusschen** [comment]
This is already implemented. Changing the volume or muting a video will apply these settings to all other videos, including those in the slideshow.

**Reply by curtwagner1984** [reply]
> This is already implemented. Changing the volume or muting a video will apply these settings to all other videos, including those in the slideshow.

Strange, I didn't notice this was happening. Is this per session, or does it persist after the browser is closed and opened again?

**Reply by michelheusschen** [reply]
These settings are stored in localStorage and should persist across multiple browsing sessions
